hack/kubevirt-nightly-test.sh

```diff
@@ -0,0 +1,147 @@
+kind: OperatorGroup
+metadata:
+  name: hco-operatorgroup
+  namespace: kubevirt-hyperconverged
+spec:
+  targetNamespaces:
+  - ${HCO_NAMESPACE}
+EOF
+
+cat <<EOF | ${CMD} create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-catalogsource-example
+  namespace: ${HCO_CATALOG_NAMESPACE}
+spec:
+  channel: ${INITIAL_CHANNEL}
+  sourceType: grpc
+  image: ${REGISTRY_IMAGE}
+  displayName: KubeVirt HyperConverged
+  publisher: Red Hat
+EOF
+
+sleep 15
+
+HCO_CATALOGSOURCE_POD=`${CMD} get pods -n ${HCO_CATALOG_NAMESPACE} | grep hco-catalogsource | head -1 | awk '{ print $1 }'`
+${CMD} wait pod $HCO_CATALOGSOURCE_POD --for condition=Ready -n ${HCO_CATALOG_NAMESPACE} --timeout="120s"
+
+CATALOG_OPERATOR_POD=`${CMD} get pods -n openshift-operator-lifecycle-manager | grep catalog-operator | head -1 | awk '{ print $1 }'`
+${CMD} wait pod $CATALOG_OPERATOR_POD --for condition=Ready -n openshift-operator-lifecycle-manager --timeout="120s"
+
+PACKAGESERVER_POD=`${CMD} get pods -n openshift-operator-lifecycle-manager | grep packageserver | head -1 | awk '{ print $1 }'`
+${CMD} wait pod $PACKAGESERVER_POD --for condition=Ready -n openshift-operator-lifecycle-manager --timeout="120s"
+
+# Creating a subscription immediately after the catalog
+# source is ready can cause delays. Sometimes the catalog-operator
+# isn't ready to create the install plan. As a temporary workaround
+# we wait for 15 seconds here. 
+sleep 15
+
+LATEST_VERSION=$(ls -d ./deploy/olm-catalog/kubevirt-hyperconverged/*/ | sort -r | head -1 | cut -d '/' -f 5);
+
+cat <<EOF | ${CMD} create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: hco-subscription-example
+  namespace: kubevirt-hyperconverged
+spec:
+  channel: ${LATEST_VERSION}
+  name: kubevirt-hyperconverged
+  source: hco-catalogsource-example
+  sourceNamespace: ${HCO_CATALOG_NAMESPACE}
+${SUBSCRIPTION_CONFIG}
+EOF
+
+# Allow time for the install plan to be created a for the
+# hco-operator to be created. Otherwise kubectl wait will report EOF.
+./hack/retry.sh 20 30 "${CMD} get subscription -n kubevirt-hyperconverged | grep -v EOF"
+./hack/retry.sh 20 30 "${CMD} get pods -n kubevirt-hyperconverged | grep hco-operator"
+
+${CMD} wait deployment ${HCO_DEPLOYMENT_NAME} --for condition=Available -n ${HCO_NAMESPACE} --timeout="1200s"
+
+${CMD} create -f ./deploy/hco.cr.yaml -n kubevirt-hyperconverged
+
+${CMD} wait -n ${HCO_NAMESPACE} ${HCO_KIND} ${HCO_RESOURCE_NAME} --for condition=Available --timeout=30m
+${CMD} wait deployment ${HCO_DEPLOYMENT_NAME} --for condition=Available -n ${HCO_NAMESPACE} --timeout="30m"
+
+Msg "check that cluster is operational"
+timeout 10m bash -c 'export CMD="${CMD}";exec ./hack/check-state.sh' 
+
+${CMD} get subscription -n kubevirt-hyperconverged -o yaml
+${CMD} get pods -n kubevirt-hyperconverged 
+
+echo "----- Image"
+${CMD} get deployments -n kubevirt-hyperconverged -o yaml | grep image | grep -v imagePullPolicy
```

Comment:
same here, and at every `-n kubevirt-hyperconverged` occurrences

Reply:
Done